### PR TITLE
invoke machine cleaner

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

File: helper.psm1 [hunk 1]
```diff
@@ -1365,6 +1365,7 @@ Function clearResults
 
 Function clearWorkdir
 {
+    python.exe jenkins/helper/clear_machine.py
```
Comment (KVS85) [inline helper.psm1:1368]: Should use `IS_JENKINS` check.

**Review by KVS85 — CHANGES_REQUESTED** [review]

File: helper.psm1 (hunk 1 above)
Comment (KVS85) [reply helper.psm1:1368]: https://github.com/arangodb/oskar/blob/master/jenkins/helper/prepareOskar.ps1

**Review by KVS85 — APPROVED** [review]

**Comment by KVS85** [comment]
https://jenkins.arangodb.biz/view/Windows/job/arangodb-ANY-windows-matrix/4083/

**Review by fceller — APPROVED** [review]

**Comment by KVS85** [comment]
https://jenkins.arangodb.biz/view/Windows/job/arangodb-ANY-windows-matrix/4088/

**Pull request merged**